service/policy/nondataresources/non_data_resources.proto

```diff
@@ -0,0 +1,212 @@
+syntax = "proto3";
+
+package policy.nondataresources;
+
+import "buf/validate/validate.proto";
+
+import "common/common.proto";
+import "policy/objects.proto";
+import "policy/selectors.proto";
+
+///
+/// Non Data Resource Groups
+///
+
+message CreateNonDataResourceGroupRequest {
+  // Required
+  string name = 1 [
+    (buf.validate.field).required = true,
+    (buf.validate.field).string.max_len = 253,
+    (buf.validate.field).cel = {
+      id: "ndr_name_format",
+      message: "Non Data Resource Group Name must be an alphanumeric string, allowing hyphens and underscores but not as the first or last character. The stored name will be normalized to lower case.",
+      expression: "this.matches('^[a-zA-Z0-9](?:[a-zA-Z0-9_-]*[a-zA-Z0-9])?$')"
+    }
+  ];
+
+  // Optional 
+  // NDR values (when provided) must be alphanumeric strings, allowing hyphens and underscores but not as the first or last character.
+  // The stored value will be normalized to lower case.
+  repeated string values = 2 [
+    (buf.validate.field).repeated = {
+      min_items: 0,
+      unique: true,
+      items: {
+        string: 
+          {
+            max_len: 253,
+            pattern: "^[a-zA-Z0-9](?:[a-zA-Z0-9_-]*[a-zA-Z0-9])?$"
+          }
+      },
+    }
+  ];
+
+  // Optional
+  // Common metadata
+  common.MetadataMutable metadata = 100;
+}
+message CreateNonDataResourceGroupResponse {
+  policy.NonDataResourceGroup group = 1;
+}
+
+message GetNonDataResourceGroupRequest {
+  // Required
+  string id = 1 [
+    (buf.validate.field).string.uuid = true
+  ];
+}
+message GetNonDataResourceGroupResponse {
+  policy.NonDataResourceGroup group = 1;
+}
+
+message ListNonDataResourceGroupRequest {
+  // Optional
+  optional string group_id = 1 [
```

Comment:
`optional` outputs a `*string` field in the generated code, which is way simpler than our current optional UUID approach elsewhere: https://github.com/opentdf/platform/blob/main/service/policy/resourcemapping/resource_mapping.proto#L19-L24

...but TBD on whether the DX takes a hit

Reply:
Removed in favor of consistency with the other protos and eventual adoption of https://buf.build/docs/protovalidate/schemas/predefined-rules/